script.js

```diff
@@ -1,4 +1,245 @@
-var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+//define variables 
+var gameMode = prompt(
+  "Welcome to a game of Beat that! There are 2 modes to explore: Lowest, and/or Auto. Type 'lowest' to turn lowest on, and 'lowest off' to turn lowest off. Type 'auto' to turn auto on, and 'auto off' to turn auto off. You can do this anytime during the game. If you want to explore the normal mode, just leave the input blank and press Ok."
+);
+
+
+
+var GAME_MODE_DICE_ROLL = 'GAME_MODE_DICE_ROLL';
+var GAME_MODE_CHOOSE_DICE_ORDER = 'GAME_MODE_CHOOSE_DICE_ORDER'; 
+
+var lowest = false; 
+
+var auto = false; 
+
+var gameMode = GAME_MODE_DICE_ROLL; 
+
+//define current player
+var currentPlayer = 1;
+
+//define players' dice rolls
+var player1DiceRoll = [];
+var player2DiceRoll = [];
+
+//define player's choice
+var player1Choice = '';
+var player2Choice = '';
+
+//keep log of players' scores
+var player1Score = []; 
+var player2Score = [];
+
+var player1Combined = function (){
+  var sum1= 0;
```

Comment:
please use a more descriptive name for variables. the sum1 doesn't tell me what sum it is. When you come back to this code in the future it might be confusing for you.